### PR TITLE
Func _WD_Startup() refactoring

### PR DESCRIPTION
**Pull request by mlipok**
### Proposed changes

Describe the big picture of your changes here to communicate to the maintainers why we should accept this pull request.<br>
Please ensure you have read and noticed the checklist below.

### Checklist

Put an `x` in the boxes that apply. If you're unsure about any of them, don't hesitate to ask. We are here to help!<br>
This is simply a reminder of what we are going to look for before merging your code.

- [x] I have read and noticed the [CODE OF CONDUCT](https://github.com/Danp2/au3WebDriver/blob/master/docs/CODE_OF_CONDUCT.md) document
- [x] I have read and noticed the [CONTRIBUTING](https://github.com/Danp2/au3WebDriver/blob/master/docs/CONTRIBUTING.md) document
- [ ] I have added necessary documentation or screenshots (if appropriate)

### Types of changes

Please check `x` the type of change your PR introduces:

- [ ] Bugfix (change which fixes an issue)
- [ ] Feature (change which adds functionality)
- [ ] Code style update (formatting, renaming)
- [x] Refactoring (functional, structural)
- [ ] Documentation content changes
- [ ] Other (please describe)

### What is the current behavior?

There is message put into console even when `$_WD_DEBUG_None` is set
Full info is put into console only when `$_WD_DEBUG_Info` is set

### What is the new behavior?

There is no message put into console even when `$_WD_DEBUG_None` is set.
Full info is put into console when `$_WD_DEBUG_Info`  is set but also when error occured and `$_WD_DEBUG_Error` is set.

### Additional context
None

### System under test
Not related

### CODE REVIEW

**Comment by Danp2** [comment]
When `$_WD_DEBUG = $_WD_DEBUG_Error`, the expectation is to only output error related messages. That no longer occurs with this change. Also, when line `If $_WD_DRIVER_DETECT And $iPID Then` is True, then none of the basic logging takes place because it now falls into the Else portion of the code. 

I'm not sure that I can agree with the direction you are headed with this.

**Comment by mlipok** [comment]
> When `$_WD_DEBUG = $_WD_DEBUG_Error`, the expectation is to only output error related messages. That no longer occurs with this change

When Error occurs in this function _WD_Startup() 
I think that it is good to have full information about what values were used and how the environment looks. 
Therefore having additionall information in case of 
```autoit
If ..... Or ($_WD_DEBUG = $_WD_DEBUG_Error And $bErrorOccured) Then
```

should be good idea.

**Comment by mlipok** [comment]
> Also, when line `If $_WD_DRIVER_DETECT And $iPID Then` is True, then none of the basic logging takes place because it now falls into the Else portion of the code.

The current (before my changes) is:
```autoit
	If $_WD_DRIVER_DETECT And $iPID Then
		__WD_ConsoleWrite($sFuncName & ": Existing instance of " & $sFile & " detected!")
	Else
		$iPID = Run($sCommand, "", ($_WD_DEBUG = $_WD_DEBUG_Info) ? @SW_SHOW : @SW_HIDE)

		If @error Or ProcessWaitClose($iPID, 1) Then
			Return SetError(__WD_Error($sFuncName, $_WD_ERROR_GeneralError, "Error launching web driver!"), 0, 0)
		EndIf
	EndIf

	Return SetError($_WD_ERROR_Success, 0, $iPID)
EndFunc   ;==>_WD_Startup
```

My question is:
The current (before my changes) is why in case of `If $_WD_DRIVER_DETECT And $iPID Then`  function returns `$_WD_ERROR_Success` ?
```autoit
	If $_WD_DRIVER_DETECT And $iPID Then
		__WD_ConsoleWrite($sFuncName & ": Existing instance of " & $sFile & " detected!")
	Else
		..........
	EndIf

	Return SetError($_WD_ERROR_Success, 0, $iPID)
EndFunc   ;==>_WD_Startup
```
Is this because of `ATTACH` feature ?

**Comment by mlipok** [comment]
> Also, when line `If $_WD_DRIVER_DETECT And $iPID Then` is True, then none of the basic logging takes place because it now falls into the Else portion of the code.

Take a look on: [Fix1](https://github.com/Danp2/au3WebDriver/pull/257/commits/06ed7b4dbd97e3418e28c04cdf0c9e81c8df7661)

File: wd_core.au3 [hunk 1]
```diff
@@ -1259,55 +1259,54 @@ Func _WD_Startup()
 	$sFile = __WD_StripPath($_WD_DRIVER)
 	$iPID = ProcessExists($sFile)
 
+	Local $bErrorOccured = False
```
Comment (mlipok) [inline wd_core.au3:1262]: I plan in next step change `$bErrorOccured` to commonly used `$iErr`

**Comment by Danp2** [comment]
> Is this because of ATTACH feature ?

Not specifically. From the variable declaration -- "Don't launch new driver instance if one already exists"

File: wd_core.au3 (hunk 1 above)
Comment (Danp2) [reply wd_core.au3:1262]: I haven't tested it, but this appears to be more inline with my expectations.

Comment (mlipok) [reply wd_core.au3:1262]: Before I do next step I will wait on your pre-aproval

Comment (Danp2) [reply wd_core.au3:1262]: If your "next step" is simple to rename the variable, then I would say it's fine to proceed. If it's something more complex, then you may want to provide further details here before proceeding.

Comment (mlipok) [reply wd_core.au3]: Done

**Review by Danp2 — APPROVED** [review]

**Pull request merged**